support/capabilities/hosted_control_plane_capabilities.go

```diff
@@ -11,6 +11,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
+func IsCapabilityEnabled(capabilities *hyperv1.Capabilities, capability hyperv1.OptionalCapability) bool {
+	if capabilities == nil {
+		return true
+	}
+	enabled := true
+	for _, disabledCap := range capabilities.Disabled {
+		if disabledCap == capability {
+			enabled = false
+		}
+	}
+	return enabled
```

Comment:
nit: we can return early here
```suggestion
	for _, disabledCap := range capabilities.Disabled {
		if disabledCap == capability {
			return false
		}
	}
	return true
```